server/form-pages/apply/offence-information/previous-unspent-convictions/unspentConvictions.test.ts

```diff
@@ -113,9 +113,11 @@ describe('UnspentConvictions', () => {
     itShouldHavePreviousValue(new UnspentConvictions({}, application), 'any-previous-convictions')
 
     describe('errors', () => {
-      it('returns empty object', () => {
+      it('returns an error when no unspent convictions are added', () => {
         const page = new UnspentConvictions({}, application)
-        expect(page.errors()).toEqual({})
+        expect(page.errors()).toEqual({
+          convictionsList: 'Unspent convictions must be added to the application'
+        })
```

Comment:
again a design rather than technical point, but the behaviour in the video, where you can go into this page, remove all existing unspent convictions, save and get this error, seems a bit user unfriendly to me. Do you think it would be better to have something to the effect of "Unspent convictions must be added to the application. If there are no relevant unspent convictions, return to the previous unspent convictions page to change your answer" (with a link)

Reply:
I think this could be a good candidate for post-launch. As this is currently in-line with the pattern elsewhere, this is good enough for private beta.